R/github.R

```diff
@@ -64,9 +64,21 @@ github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
   if (identical(pat, "")) return(NULL)
```

Comment:
You are correct, fixed with 199b0c2eab7e67af06586d1c2eda8fe34add2f36. Thank you for catching that I clearly was not thinking it through correctly.